src/scripts/validation.js

```diff
@@ -75,13 +80,15 @@ const setEventListeners = (formEl, config) => {
       toggleButtonState(inputList, buttonElement, config);
     });
   });
+
+  formEl.addEventListener("submit", function () {
+    disableButton(buttonElement, config);
```

Comment:
You need to `disable` the button only after **resetting** the form when you have submitted a card  or changed the avatar, to be able to click it again if there is a server error during the api handling

And that's why I can't delete 2 cards in a row (the button gets disabled afte submisson rather than after resetting the form)


Please, change the `event` in line 84. It should be `reset` instead of `submit`


```js
  formEl.addEventListener("reset", function () {  // like this
    disableButton(buttonElement, config);
  });

```